code/modules/mob/living/simple_animal/abnormality/zayin/pile_of_mail.dm

```diff
@@ -0,0 +1,303 @@
+	pickup_sound =  'sound/items/handling/paper_pickup.ogg'
+	gender = NEUTER
+	color = "white"
+	var/mob/living/carbon/receiver = null
+
+//this is using the parcel. you either go insane or get special ego
+/obj/item/parcelself/attack_self(mob/living/carbon/human/user)
+	if(receiver != user)
+		to_chat(user,"<span class='notice'>It feels wrong to try to open someone else's package.</span>")
+		return
+	if(prob(50))
+		new /obj/item/ego_weapon/mail_satchel(get_turf(user))
+		to_chat(user,"<span class='nicegood'>As you unravel the parcel a satchel falls out.</span>")
+	else
+		new /obj/item/clothing/suit/armor/ego_gear/letter_opener(get_turf(user))
+		to_chat(user,"<span class='nicegood'>As you unravel the parcel an outfit falls out.</span>")
+	qdel(src)
+	return
+
+/obj/item/parcelself/New(loc,mob/living/carbon/human/user, ...)
+	. = ..()
+	desc = "This parcel seems to be addressed to [user.name]. Where did it come from..?"
+	receiver = user
+
+/obj/item/mailpaper
+	name = "Mail"
+	desc = "One of the letters from the pile slowly waves in the air."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mail"
+	inhand_icon_state = "mail"
+	worn_icon_state = "mail"
+	drop_sound = 'sound/items/handling/paper_drop.ogg'
+	pickup_sound =  'sound/items/handling/paper_pickup.ogg'
+	gender = NEUTER
+	color = "white"
+
+/obj/item/mailpaper/pipebomb
+	name = "Suspicious letter"
+	desc = "A letter addressed to a scratched out name, by... Neco-Arc?"
+
+/obj/item/mailpaper/pipebomb/attack_self(mob/living/carbon/human/user)
+	to_chat(user,"<span class='warning'>What is this... Is this a pipebomb!?</span>")
+	sleep(10)
+	//add glitterbomb sfx, maybe a toot horn or something funny
+	for(var/direction in GLOB.alldirs)
+		var/turf/dispense_turf = get_step(src, direction)
+		new /obj/effect/decal/cleanable/glitter(dispense_turf)
+	qdel(src)
+
+/obj/item/mailpaper/junk
+	var/JUNKMAIL = list(list("A letter from RobustCO","The outside reads 'Improve your robustness today! Read our letter now!"),list("Suspicious message","Ever felt like you need a boost? Get swole today!"))
+
+/obj/item/mailpaper/junk/Initialize()
+	. = ..()
+	var/temp = pick(JUNKMAIL)
+	name = temp[1]
+	desc = temp[2]
+
+/obj/item/mailpaper/junk/attack_self(mob/living/carbon/human/user)
+	to_chat(user,"<span class='notice'>This was a waste of time....</span>")
+	user.adjustSanityLoss(5)
+	qdel(src)
+
+/obj/item/mailpaper/hatred
+	name = "A hastily written angry letter"
+	desc = "IHATEYOUIHATEYOUIHATEYOUIHATEYOUIHATEYOU."
+
+/obj/item/mailpaper/hatred/attack_self(mob/living/carbon/human/user)
+	to_chat(user,"<span class='warning'>The pages are filled with scribbles and threats...</span>")
+	user.adjustSanityLoss(5)
+	qdel(src)
+
+/obj/item/mailpaper/instinct
+
+/obj/item/mailpaper/instinct/New(loc,mob/living/carbon/human/user, ...)
+	. = ..()
+	desc = "A past letter from a friend addressed to [user.name]. The thought of such conduct makes you feel happy, albeit bittersweet."
```

Comment:
```suggestion
	desc = "A past letter from a friend addressed to \a [user]. The thought of such conduct makes you feel happy, albeit bittersweet."
```
`[user.name]` here is unnecessary and could cause issues with special names like improper nouns. Using `\a [user]` instead will automatically insert the name plus an article, if needed, so you'd get `addressed to a stranger` and `addressed to Catt` for `stranger` and `Catt` respectively.